lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -16,6 +16,7 @@ package servicers
 import (
 	"context"
 	"fmt"
+	"hash/fnv"
```

Comment:
Why fnv sum? Add note somewhere on choice

Reply:
Added note on choice. Basically "use the simplest tool that suffices."